cli.js

```diff
@@ -116,11 +120,15 @@ ee.on('finish', async () => {
 	try {
 		let identity;
 		const {stdout} = await execa('security', ['find-identity', '-v', '-p', 'codesigning']);
-		if (stdout.includes('Developer ID Application:')) {
+		if (cli.flags.identity && stdout.includes(`"${cli.flags.identity}"`)) {
```

Comment:
Actually, I don't think it should just silently fall back to automatic if the passed in identity does not exist. It should instead error with a user-friendly error message.

Reply:
In a given case we always receive `No usable identity found` error - is it fine or message should be different?

Reply:
Let's go with:

> No suitable code signing identity found